index.html

```diff
@@ -2,38 +2,70 @@
 <head>
 <title>Chef Brian Bevand</title>
 <style type="text/css">
-table, h1
-{
-	border: 0px solid black;
-	border-collapse:collapse;
-	width: 100%;
-}
-td
-{
-	border: 0px;
-	border-collapse:collapse;
-	width: 25%;
+body{ 
+    background-image: url("veggies.jpg");
+    background-position: center;
+    background-size: cover;
+	font-family: sans-serif;
+	color: white;
 }
 img {
 	max-width: 100%;
 	height: auto;
+	opacity: 100%;
+}
+.title {
+	font-family:impact;
+	color: white;
+	font-size: 60px;
+}
+.extra-text {
+  display: inline-block;
```

Comment:
inconsistent indentation

Reply:
same. like food. 
So how do you customize parts of the site? what if I want a style for a text ??

Reply:
I was putting classes for everything. I can put it whith properties right?